libazureinit/src/http.rs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+use reqwest::StatusCode;
+
+// Set of StatusCodes that should be retried,
+// e.g. 400, 404, 410, 429, 500, 503.
```

Comment:
Since this is a public constant, using `///` instead of `//` will cause it to be rendered in the HTML docs, and I think we should make an effort to have all public symbols documented. You can see how things look locally with `cargo doc -p libazureinit --no-deps`.